build.ps1

```diff
@@ -1,4 +1,3 @@
-#!/usr/bin/env pwsh
```

Comment:
This needs to be here for Linux/MacOS support

Reply:
I dont think I changed this file.

Reply:
This is still getting removed